### PR TITLE
Adding user creation to user profile timeline

### PR DESCRIPTION
**Pull request by superalex**

### CODE REVIEW

File: taiga/timeline/management/commands/rebuild_timeline_for_user_creation.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import gc
+
+class BulkCreator(object):
+    def __init__(self):
+        self.timeline_objects = []
+        self.created = None
+
+    def createElement(self, element):
+        self.timeline_objects.append(element)
+        if len(self.timeline_objects) > 1000:
```
Comment (jespino) [inline taiga/timeline/management/commands/rebuild_timeline_for_user_creation.py]: What happens when the timeline_objects length is less than 1000? For example, I insert 3 timeline entries, this is never persisted?

Comment (superalex) [inline taiga/timeline/management/commands/rebuild_timeline_for_user_creation.py]: Ops!, I have to fix this part! :dancer:

Comment (superalex) [inline taiga/timeline/management/commands/rebuild_timeline_for_user_creation.py]: Fixed now!

**Comment by jespino** [comment]
The tests are failing because you hasn't the database migration for remove the not-null project foreign key restriction.

**Comment by superalex** [comment]
@jespino you are right, I forgot adding it :(

**Pull request merged**